packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -830,24 +830,24 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       goGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}go-client-getting-started`,
       httpApis: `${SERVERLESS_ELASTICSEARCH_DOCS}http-apis`,
       httpApiReferences: `${SERVERLESS_ELASTICSEARCH_DOCS}http-apis`,
-      jsApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}nodejs-apis-getting-started`,
-      jsGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}nodejs-apis-getting-started`,
-      phpApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}php-apis-getting-started`,
-      phpGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}php-apis-getting-started`,
-      pythonApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}python-apis-getting-started`,
-      pythonGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}python-apis-getting-started`,
-      pythonReferences: `${SERVERLESS_ELASTICSEARCH_DOCS}python-apis-getting-started`,
-      rubyApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}ruby-apis-getting-started`,
-      rubyGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}ruby-apis-getting-started`,
+      jsApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}nodejs-client-getting-started`,
+      jsGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}nodejs-client-getting-started`,
+      phpApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}php-client-getting-started`,
+      phpGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}php-client-getting-started`,
+      pythonApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}python-client-getting-started`,
+      pythonGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}python-client-getting-started`,
+      pythonReferences: `${SERVERLESS_ELASTICSEARCH_DOCS}python-client-getting-started`,
+      rubyApiReference: `${SERVERLESS_ELASTICSEARCH_DOCS}ruby-client-getting-started`,
+      rubyGettingStarted: `${SERVERLESS_ELASTICSEARCH_DOCS}ruby-client-getting-started`,
     },
     serverlessSearch: {
       integrations: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-your-data`,
-      integrationsLogstash: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-data-through-integrations-logstash`,
-      integrationsBeats: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-data-through-integrations-beats`,
-      integrationsConnectorClient: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-data-through-integrations-connector-client`,
-      gettingStartedExplore: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started#explore`,
-      gettingStartedIngest: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started#ingest`,
-      gettingStartedSearch: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started#search`,
+      integrationsLogstash: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-data-through-logstash`,
+      integrationsBeats: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-data-through-beats`,
+      integrationsConnectorClient: `${SERVERLESS_ELASTICSEARCH_DOCS}ingest-your-data`,
+      gettingStartedExplore: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started`,
+      gettingStartedIngest: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started`,
+      gettingStartedSearch: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started`,
```

Comment:
>      gettingStartedSearch: `${SERVERLESS_ELASTICSEARCH_DOCS}get-started`,

It seems odd to have three keywords that point to the same URL when only one of them is used anywhere. I'd vote for removing the unused duplicates. I can append a commit to the PR to do so if you agree.

Reply:
++

There are several issues in these files that are creating technical debt. I can follow up with an engineer to help clean those up.